README.md

```diff
@@ -91,35 +101,33 @@ proper path to its configuration file. For example, it may look like:
 elasticsearch --config=/Users/[YOUR MAC OSX USERNAME]/homebrew/opt/elasticsearch/config/elasticsearch.yml
 ```
 
-### 3. Launch Sheer to serve the site
-
-To work on the app you will need Sheer running to compile the templates.
+### 3. Load Indexes & Launch Site
 To do this, run the following:
 
 ```bash
 # Use the cfgov-refresh virtualenv.
 workon cfgov-refresh
 
-# cd into the /dist/ directory.
-cd ./dist
+# cd into the /cfgov/v1/jinja2/v1/ directory.
+cd cfgov/v1/jinja2/v1
 
 # Index the latest content from the API output from a WordPress and Django back-end.
 # **This requires the constants in INSTALL - Configuration to be set**
 sheer index
 
-# Start sheer.
-sheer serve --debug
+# From the Project root, start server.
+python cfgov/manage.py runserver
 ```
 
-To view the site browse to: <http://localhost:7000>
+To view the site browse to: <http://localhost:8000>
```

Comment:
This will need to be updated elsewhere in the project https://github.com/cfpb/cfgov-refresh/search?utf8=%E2%9C%93&q=7000